### PR TITLE
add triage team members

### PR DESCRIPTION
**Pull request by vdeturckheim**
Following https://github.com/nodejs/security-wg/issues/73

@nodejs/security-wg please +1 or -1 this. I'd like to have this merged in order to invite members to the team. Once this is done, we will need to spread the word and handle the few issues already reported on HackerOne.

cc @sam-github @cjihrig @bengl @sam-github @brycebaril

### CODE REVIEW

**Comment by cjihrig** [comment]
+1 from me, although I think we should add the names in alphabetical order (probably by GitHub username, as is done in nodejs/node).

**Comment by vdeturckheim** [comment]
fair point @cjihrig ... fixed

File: processes/third_party_vuln_process.md [hunk 1]
```diff
@@ -92,4 +92,8 @@ as the [Node.js Security Team](https://github.com/nodejs/security-wg/blob/master
 Members of the security teams should indicate that they accept the privacy policies 
 by PRing their acceptance to this file:
```
Comment (gibfahn) [inline processes/third_party_vuln_process.md]: This suggests that people should raise individual PRs to add themselves to the list. I have no issue with us not doing that this time, but maybe @bengl @brycebaril @cjihrig and @sam-github should comment in this PR with their acceptance?

Comment (vdeturckheim) [reply processes/third_party_vuln_process.md]: @gibfahn you're actually right! My bad.

@bengl @brycebaril @cjihrig and @sam-github : I think the safest way is to close current PR and have you open new ones.

As no-one objected on this current one, I think we will be safe to merge your individual PRs immediately.

Sorry about this process mistake from me.

Comment (sam-github) [reply processes/third_party_vuln_process.md]: I don't think I should be involved in the thirdparty triage at this time - I don't have the time. I do accept the privacy policies, though!

**Pull request closed without merging**